src/sections/quiz/quiz-body.js

```diff
@@ -3,35 +3,119 @@ import {SvgIcon} from "@mui/material";
 import Container from "@mui/material/Container";
 import Box from "@mui/material/Box";
 import Logo from "../../components/svg-color/Logo.svg";
-import {Lora} from '@next/font/google';
-import Grid from '@mui/material/Grid';
+import {Lora} from "@next/font/google";
+import Grid from "@mui/material/Grid";
+import useSWR from "swr"
+import { useState } from "react";
+import { useSWRConfig } from "swr"
 
+const fetcher = (...args) => fetch(...args).then(res => res.json())
+const post_fetcher = (...args) => fetch(...args).then(res => res.json())
 const lora = Lora({preload: false});
 
-// function VariantButton(text) {
-function VariantButton() {
+
+function QuestionList() {
```

Comment:
Function name should start from lower-case letter. The default naming convention is camelCase.
React components start from an upper-case letter to distinguish from regular functions. Naming convention is PascalCase.